tools/RunVersionController.ps1

```diff
@@ -383,10 +383,15 @@ switch ($PSCmdlet.ParameterSetName)
         dotnet $PSScriptRoot/../artifacts/VersionController/VersionController.Netcore.dll
 
         Bump-AzVersion
+    }
+    
+    # Each release needs to update AzPreview.psd1 and dotnet csv
+    {$PSItem.StartsWith("Release")}
```

Comment:
$PSItem (ParameterSetName) always starts with "Release". Why not just pull line 392-395 to the outest layer?

Reply:
I mean, if we decide to refresh AzPreview every time we run this script, it makes more sense to me to write it like:
```powershell
switch (parameter set) {
...
...
}
Generate-AzPreview
```